mysql-test/suite/innodb/t/bug84563.test

```diff
@@ -0,0 +1,35 @@
+--source include/have_innodb.inc
+
+SET GLOBAL innodb_lock_wait_timeout = 1;
+SET GLOBAL innodb_print_lock_wait_timeout_info = true;
+
+CREATE TABLE t (i INT) ENGINE = InnoDB;
+INSERT INTO t (i) VALUES(1);
+
+START TRANSACTION;
+SELECT * FROM t WHERE i = 1 LOCK IN SHARE MODE;
+
+--connect(con0, localhost, root,,)
+START TRANSACTION;
```

Comment:
may be, in this case, just
```
START TRANSACTION;
--error ER_LOCK_WAIT_TIMEOUT
DELETE FROM t WHERE i = 1;
```
without `--send / --reap`

Reply:
Yes, sure. I used one more connection to get additional information from I_S and PFS, and forgot to remove this --send/--reap when this additional connection was removed. Fixed.